build/index.html

```diff
@@ -119,12 +119,12 @@
         </div>
     </div>
     <div id="world"></div>
-    <script src="js/three.min.js"></script>
-    <script src="js/lib/create_segments.js"></script>
+    <script src="js/three.min.js"></script> //Three muss zuerst!
```

Comment:
Kommentare in HTML werden mit "**<!-- KOMMENTAR -->**" erstellt. So würde man den Text theoretisch auf der Webseite sehen, sofern kein Content darüber liegt.